source/_integrations/zha.markdown

```diff
@@ -575,58 +575,61 @@ After restoring a Home Assistant backup, you can reconfigure ZHA or migrate to a
+2. Under **Network settings**, select **Migrate adapter**.
 3. Reconfiguration of ZHA will start. Select **Submit**.
-4. Under **Migrate or re-configure**, select **Migrate to a new radio**.
-5. Select **Submit** to begin.
-    - An automatic backup will be performed, the Zigbee Coordinator radio will be reset, and the backup will be automatically restored.
-    - For combined Z-Wave and Zigbee radios, like the HUSBZB-1 adapter, only the Zigbee radio portion is reset.
-    - You may now unplug the old adapter, or you may leave the old radio adapter plugged in (for example, if the adapter is a combined Z-Wave adapter).
-6. Plug in the new Zigbee Coordinator radio adapter.
-    - Select **Submit** after confirming the new Zigbee Coordinator radio adapter is properly connected.
-    - To minimize interference:
-      - Use a USB extension cable,
-      - Use a USB 2.0 port or a powered USB 2.0 hub,
-      - Keep the Zigbee stick away from USB 3.0 devices.
-7. Start the backup restore process:
-    - Select the new Zigbee radio from the list of serial ports and select **Next**.
-    - A migration can be resumed if a reboot is required, such as when troubleshooting or if new hardware is plugged in.
-      - To resume, go to **Network Settings**, select **Re-configure the current radio**, choose the new radio and proceed.
-8. Under **Network Formation**, select **Restore an automatic backup**.
-9. Under **Restore Automation Backup**, choose the latest automatic backup and select **Submit**.
-10. If the new radio requires overwriting the IEEE address (the unique MAC address), you will see the prompt for **Overwrite Radio IEEE Address**.
-    - Check the **Permanently replace the radio IEEE address** box and click **Submit**.
-      - Selecting this option is required for the migration process to complete successfully.
-      - Overwriting the IEEE address may take a while.
-    - Both the old and new Zigbee Coordinators will now have the same Zigbee IEEE address.
-    - You should not operate the old adapter in the same area unless you change its Zigbee IEEE address.
-    - If you do not migrate the Zigbee IEEE address from the old Zigbee Coordinator radio, you will have to reconnect many of your devices to keep them working.
-11. Finally, a **Success!** message should pop up with information that all options were successfully saved.
-    - Select **Finish** to confirm.
+   - An automatic backup will be performed, the Zigbee adapter will be reset, and the backup will be automatically restored.
+   - For combined Z-Wave and Zigbee adapters, like the HUSBZB-1 adapter, only the Zigbee radio portion is reset.
+   - You may now unplug the old adapter, or you may leave the old adapter plugged in (for example, if the adapter is a combined Z-Wave adapter).
+4. Under **Migrate or re-configure**, select **Migrate to a new adapter**.
+5. Plug in the new Zigbee adapter.
```

Comment:
With the new flow, the new adapter should be plugged in since the start now. Or at least one step before the screen with the serial port selection. Otherwise, it won't show there.